cmake/OpenLocoUtility.cmake

```diff
@@ -79,6 +79,7 @@ function(loco_target_compile_link_flags TARGET)
         # -Warray-bounds
         # compilers often get confused about our memory access patterns, disable some of the warnings
         -Wno-array-bounds
+        -Wno-deprecated-declarations
```

Comment:
I would move this down next to -fno-char8_t and add a little note that this is disabled due to using no-char8_t.